### ISSUE TITLE
search results show duplicate items with symlinks

### ISSUE THREAD

**Issue by dirk-thomas** [atom/find-and-replace#711]
In the following example the project find results view shows the same items multiple times:

* open a project which contains symlinked content
* do a project wide search (at this point the results are ok)
* select one of the results which is in a symlinked location
* once the file is being opened in a new tab a new search result appears (which is a duplicate of the item you clicked on just with the real path of the file)

**Comment by Rockster160** [comment]
👍 Updated recently and am running into the same Bug.
Currently running 1.12.2.
I don't recall seeing this issue previously.

**Comment by Ben3eeE** [comment]
@Rockster160 are you on a network mapped drive? https://github.com/atom/atom/issues/13104 might be the issue you are seeing.

**Comment by Rockster160** [comment]
Ah. That may actually be it. Recently started using a VPN as well. Sorry for mis-categorizing!

**Comment by dirk-thomas** [comment]
I am not using a network mapped drive but a single local partition. So it it seems to be not only happening with network mapped drives.

**Comment by Ben3eeE** [comment]
@dirk-thomas Yeah this is still an issue. Just wanted to steer @Rockster160 to the correct issue since he mentioned it was recent while this is not.

**Comment by kyleshevlin** [comment]
So, if you open the inspector, the search results aren't *true* duplicates. The paths to the two identical search results are different, the first will be your symlinked path, the second will be the full absolute path. Perhaps others have noticed that already, but I figure this information could be helpful.

**Comment by jblandry** [comment]
I think I have the exact reversed pproblem here → https://github.com/atom/find-and-replace/issues/445#issuecomment-375735079
